setup.sh

```diff
@@ -22,6 +22,7 @@ mount /r/var/lib/lxc/android/android-rootfs.img /s
 # Set udev rules
 ui_print "Setting udev rules";
 cat /s/ueventd*.rc /vendor/ueventd*.rc | grep ^/dev | sed -e 's/^\/dev\///' | awk '{printf "ACTION==\"add\", KERNEL==\"%s\", OWNER=\"%s\", GROUP=\"%s\", MODE=\"%s\"\n",$1,$3,$4,$2}' | sed -e 's/\r//' > /r/etc/udev/rules.d/70-$VENDOR_DEVICE_PROP.rules;
+sed -i '/GROUP="radio"\|GROUP="root"\|GROUP="system"/!s/GROUP="\([^"]*\)"/GROUP="android_\1"/' /r/etc/udev/rules.d/70-$VENDOR_DEVICE_PROP.rules;
```

Comment:
can this be added in pipe on the line before? :thinking:

Reply:
Made changes!